src/controllers/midi/midicontroller.h

```diff
@@ -10,15 +10,16 @@
 
 class MidiOutputHandler;
 
-class MidiInputHandlerController : public QObject {
+class MidiInputHandlerController final : public QObject {
```

Comment:
lets also call this `MidiInputHandleJSProxy` to follow the naming scheme of the other classes exposed to JS. In the long-term we should probably just put these in a namespace, but thats a refactoring for another PR.